protobuf/src/test/scala/magnolify/protobuf/test/ProtobufTypeSpec.scala

```diff
@@ -109,3 +131,16 @@ case class NestedNoOption(
   r: Required,
   l: List[Required]
 )
+
+case class NullableNoneValue(
+  @noneValue(true) b: Option[Boolean],
```

Comment:
Is there a better name? `@none`/`@asNone`?

Reply:
I like `asNone`? maybe `valueAsNone` or `defaultAsNone`?

Reply:
I'm also leaning towards `asNone`, that way `None` is also properly capitalized :)
`default` can be confused with scala arg default value.